Merge Java profiler setup steps (#11144)
DataDog/documentation: content/en/tracing/profiler/enabling.md

    java -javaagent:dd-java-agent.jar -Ddd.profiling.enabled=true -XX:FlightRecorderOptions=stackdepth=256 -jar <YOUR_SERVICE>.jar <YOUR_SERVICE_FLAGS>
    ```

> 4. **Recommendation**: Specify the `service` and `version` so you can slice and dice your profiles across these dimensions. [Use environment variables](#environment-variables) to set the parameters.

Reviewer: Thanks for improving this!

We should take one more step, which is to merge this with step 3 or otherwise structure it so when we show an example snippet of code, it matches our recommendations. Otherwise, again, it's sort of like:

User: "Ok this page has a bunch of words, skimming... Oh great, a code snippet, let me copy/paste that and fill in my values."
Us: "Gotcha! You didn't include service or version (or env) so your experience won't be as cool!"
User: "But, but, I used the code snippet you showed me!"
Lurker: <reacts with :itsatrap:>

:-)

Or maybe they notice step 4 in time but rather than seeing exactly how to use these tags, they have to jump elsewhere to learn.

You can see how we get this right on the other language tabs- https://docs.datadoghq.com/tracing/profiler/enabling/

PS We should also show env.
PPS You can set these with system properties or env vars. You can see in Python and Ruby how we show snippets for both versions, and in Go where we prefer one up top and show the alternate below.

Author: > PPS You can set these with system properties or env vars. You can see in Python and Ruby how we show snippets for both versions, and in Go where we prefer one up top and show the alternate below.
There is no way to do that in Java. The Profiler can only be enabled when launching the JVM, not once the JVM has already started.

> 4. **Optional** - Datadog strongly recommends that you specify the `service` and `version` as this gives you the ability to slice and dice your profiles across these dimensions. [Use environment variables](#environment-variables) to set the parameters.

Reviewer 2: ```suggestion
4. **Recommendation**: Specify the `service` and `version` so you can to slice and dice your profiles across these dimensions. [Use environment variables](#environment-variables) to set the parameters.
```
